app/models/user.rb

```diff
@@ -5,10 +5,16 @@ class User < ActiveRecord::Base
 
   has_many :recommended_items
   has_many :television_shows, through: :recommended_items
-
   has_many :following_relationships,
     class_name: "Relationship", foreign_key: :follower_id
   has_many :following, through: :following_relationships, source: :followed
+  has_many :follower_relationships,
+    class_name: "Relationship", foreign_key: :followed_id
+  has_many :followers, through: :follower_relationships, foreign_key: :follower
+
+  def self.recommended_to_follow(except:)
+    where.not(id: except).first(3)
```

Comment:
Good call. I used `except:` since it's a class method but this could easily be an instance method instead so we can reference `self`.

d9d259e81f1b13689416e09da21e9aa8c01d7a2f